modules/tinymce/src/plugins/searchreplace/main/ts/core/Actions.ts

```diff
@@ -20,6 +20,9 @@ export interface SearchState {
   wholeWord: boolean;
 }
 
+// Copied from UnicodeData in polaris, as it's not exported. Maybe it should be?
```

Comment:
not duplicating complex regex definitions is generally a good idea 🤔

Reply:
One big issue is that Polaris really isn't setup for tree-shaking, as it puts all the strings/regexes in a single object.

Reply:
Okay, I've exposed it from Polaris instead in a way that I believe makes sense 🤷‍♀